test.py

```diff
@@ -0,0 +1,84 @@
+import os
```

Comment:
this should be a unit test or removed from the PR

Reply:
included by mistake, will delete.